File: 8_Dictionaries/javascript/scratch.js
import { DsDictionary } from "./DsDictionaryModule.js";

const d = new DsDictionary([
  ["name", "Ada"],
  ["role", "Engineer"],
]);

console.log(d.constructor.name);
console.log(String(d));

d.set("language", "Python");
console.log(String(d));

console.log("keys:", [...d.keys()]);
console.log("values:", [...d.values()]);
console.log("items:", [...d.items()]);

File: 8_Dictionaries/python/scratch.py
from DsDictionaryModule import DsDictionary

d = DsDictionary([("name", "Ada"), ("role", "Engineer")])
print(type(d))
print(d)

d["language"] = "Python"
print(d)

print("keys:", list(d.keys()))
print("values:", list(d.values()))
print("items:", list(d.items()))
